src/services/validatorSchema.ts

```diff
@@ -0,0 +1,17 @@
+import * as yup from 'yup';
+
+export const inventoryItemSchema = yup.object({
+    user_id: yup.string().defined("All fields must be completed"),
+    item_id: yup.number().positive().defined("All fields must be completed"),
+});
+
+export const userSchema = yup.object({
+    user_id: yup.number().positive().defined("All fields must be completed")
+});
+
+export const warehouseItemSchema = yup.object().shape({
+    name: yup.string().matches(/[A-Za-z]/, "Only alphabet words").defined("All fields must be completed"),
```

Comment:
I think the name could be anything. For simplicity, we use a alpha numeric string in our demo, but I was thinking that editors could use anything they liked, even serial numbers like "23jh4j-k5h4Y" if they liked it. So just testing if it's a string should be enough :) also, the error message should be explicit: "name should be provided"